aiida/work/job_calcs.py

```diff
@@ -0,0 +1,235 @@
+import contextlib
+from future.utils import iteritems, itervalues
```

Comment:
FYI: In the Python 3 PR I dropped `future` completely in favor of `six`, which has a `iteritems` as well. But I dropped it in most cases in favor of just using `d.items()`.

Reply:
Thanks Tiziano!  I'll convert over to six.